src/openrct2-ui/CMakeLists.txt

```diff
@@ -232,7 +232,21 @@ if(MACOS_BUNDLE)
         ZIP_URL ${TITLE_SEQUENCE_URL}
         SHA1 ${TITLE_SEQUENCE_SHA1}
     )
-    
+
+    # download opensfx and openmsx
```

Comment:
This only adds downloads to the macOS bundle.

I tried adding counterparts for Linux to `<root>/CMakeLists.txt`, but I’m running into problems there - it seems that merging three `object` folders does not work as intended.